example-configs/yi-emacs-vty-static/Main.hs

```diff
@@ -1,7 +1,14 @@
+-- This is a configuration example, that requires manual rebuild (as opposed to dynamic ones, using
+-- Dyre, which you're, probably, actually want). Here's a building example with "stack":
```

Comment:
I actually want static configs so that I can safely not think about path to ghc, available packages and am able to copy the editor binary to other machines.